### ISSUE TITLE
A release was made 3 days ago.

### ISSUE THREAD

**Issue by shaggytwodope** [linuxlite/liteusermanager#2]
A release was tagged 3 days ago, among the changes was a fix for broken functionality. 

Steps to re-produce... open try to change password for current user.

Tags are made by dev's for a reason. If work is being ignored, I'll gladly throw in my hat for this stuff.
Doing things when you feel like it does not respect other peoples efforts. A responsible person would handle control to a trusted person if they are unable to do so.

This package has been broke for months.

Offending commit: https://github.com/linuxlite/liteusermanager/commit/8ae71e6714dfbaecacbb377717be9e9757e5394a

**Comment by AdrianKoshka** [comment]
Have to agree here with @shaggytwodope...bug fixes like this shouldn't be ignored for so long, and if one doesn't have the time to do it themselves, hand over responsibility to another capable person.

**Comment by linuxlite** [comment]
Hi folks,

Nothing gets ignored, I get email notifications for everything. Time, as I have publicly explained, is the factor here. It does seem that both of you would be happier contributing your considerable talents and generosity of time to another project or projects as this is one of many issues that have come to pass in recent times. This seems like a solution that would ultimately bring happiness to all parties. If this is the case, I wish you both the best and thank you sincerely for your contributions to free software. Kind regards, Jerry.

**Comment by shaggytwodope** [comment]
You basiclly said, "I saw but didn't care to do anything"

And "To fucking bad guys, don't like it leave"

3 days with no response or action is ignoring. It's more then enough time for anyone to act on.The amount of text you typed to respond is more then the amount of commands to create a .deb file. And you  know damn well I'm right. 

You are unable to do things and yet keep pretending you are ontop of things.  I've overlooked alot of things for to long. You passive agressive responses are in bad taste. This is no team work or project when you dictate how everything is done or when you want to.

Bottom line, you ignored things and people. If you are unable to do things you really should own up to that and let go of your obsession with control. e.g. taking my projects and making them yours. You know it's wrong to copy the repos way you did. And don't have "time" to fix it. Even after giving you the best way to do it. 

Any decent person would take my work down from the project, but you wont. You stole my projects. You even package them poorly. Without a license.. violating the gpl btw. We might talk about that later when I have "time".

**Comment by linuxlite** [comment]
Hi shaggy, there's no passive aggressive approach here. I have always tried to be kind and courteous, it is my nature to be so. I desire a peaceful existence. Would you like me to remove LLCC from future LL releases?
